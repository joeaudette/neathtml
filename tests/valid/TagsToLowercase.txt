<SPAN>Test</SPAN><Span>Test2</Span>
