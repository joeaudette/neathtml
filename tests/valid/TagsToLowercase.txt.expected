<span>Test</span><span>Test2</span>
